<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>product</class>
 <widget class="QWidget" name="product">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>568</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>NSimSun</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Icon/product.png</normaloff>Icon/product.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>451</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala Text</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Show all product</string>
   </property>
   <widget class="QTableWidget" name="TBproduct">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>431</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala Text</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Product name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Product price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Product type</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>451</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala Text</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>search product</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="txtSearch"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSearch">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Icon/cloud.png</normaloff>Icon/cloud.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>261</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala Text</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>In put product</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="txtID">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtName"/>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPrice"/>
     </item>
     <item>
      <widget class="QLineEdit" name="txtType"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>90</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Product Id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Product Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Product price</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Product type</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala Text</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Commands</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="btnInsert">
       <property name="text">
        <string>Insert</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Icon/import.png</normaloff>Icon/import.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUpdate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Icon/data-processing.png</normaloff>Icon/data-processing.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>Clear</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Icon/broom.png</normaloff>Icon/broom.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Icon/delete.png</normaloff>Icon/delete.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
